lib/middleware/request_tracker.rb

```diff
@@ -166,62 +169,61 @@ def self.populate_request_queue_seconds!(env)
   def call(env)
     result = nil
     log_request = true
+    info = nil
 
     # doing this as early as possible so we have an
     # accurate counter
     ::Middleware::RequestTracker.populate_request_queue_seconds!(env)
 
     request = Rack::Request.new(env)
-
-    if available_in = rate_limit(request)
-      return [
-        429,
-        { "Retry-After" => available_in.to_s },
-        ["Slow down, too many requests from this IP address"]
-      ]
+    cookie_string = request.cookies[Auth::DefaultCurrentUserProvider::TOKEN_COOKIE].presence
```

Comment:
The current user auth provider can change so I don't think we should be referencing the `Auth::DefaultCurrentUserProvider::TOKEN_COOKIE` constant in the middleware?